packages/paste-core/components/button/__tests__/customization.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
```

Comment:
I'm getting a `tabIndex` error on the customization tests. Is there something about nesting the Button within the CustomizationProvider that makes it think I'm setting the `tabIndex` to `-2`?
```
console.error
    Warning: Failed prop type: Invalid prop `tabIndex` of value `-2` supplied to `Button`, expected one of [0,-1].
        in Button
```

Reply:
I can't see anything. Have you used the react testing library debugger to see whats going on?

Reply:
Added to the test that was causing the prop types error to expect it and now I'm not seeing the error